presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -602,7 +608,7 @@ private static DynamicFiltersResult createDynamicFilters(
         {
             Map<String, VariableReferenceExpression> dynamicFilters = ImmutableMap.of();
             List<RowExpression> predicates = ImmutableList.of();
-            if (node.getType() == INNER && isEnableDynamicFiltering(session)) {
```

Comment:
Why removing this?

Reply:
This is inside the `createDynamicFilters` function, not in `visitJoin`